docs/csharp/language-reference/language-specification/toc.md

```diff
@@ -0,0 +1,17 @@
+# [1.0 Lexical structure](lexical-structure.md)
+## [1.1 Programs](lexical-structure.md#programs)
+## [1.2 Grammars](lexical-structure.md#grammars)
+## [1.3 Lexical analysis](lexical-structure.md#lexical-analysis)
+## [1.4 Tokens](lexical-structure.md#tokens)
+## [1.5 Pre-processing directives](lexical-structure.md#pre-processing-directives)
+# [2.0 Basic concepts](basic-concepts.md)
+## [2. 1 Application Startup](basic-concepts.md#application-startup)
+## [2.2 Application termination](basic-concepts.md#application-termination)
+## [2.3 Declarations](basic-concepts.md#declarations)
+## [2.4 Members](basic-concepts.md#members)
+## [2.5 Member access](basic-concepts.md#member-access)
+## [2.6 Signatures and overloading](basic-concepts.md#signatures-and-overloading)
+## [2.7 Scopes](basic-concepts.md#scopes)
+## [2.8 Namespace and type names](basic-concepts.md#namespace-and-type-names)
+## [2.9 Automatic memory management](basic-concepts.md#automatic-memory-management)
+## [2.10 Execution order](basic-concepts.md#execution-order)
```

Comment:
I am ambivalent as well. I'm going to look at the rendered site once the build finishes, and then make a first recommendation.

Honestly, I'm leaning against them, because I don't think we can automate the section numbering.